src/schemas/base-types.yaml

```diff
@@ -35,11 +35,15 @@ uint:
   title: hex encoded unsigned integer
   type: string
   pattern: ^0x([1-9a-f]+[0-9a-f]*|0)$
+uint64:
+  title: hex encoded unsigned integer
+  type: string
+  pattern: ^0x[0-9a-f]{0,16}$
```

Comment:
Is this meant to encode the `QUANTITY` type from the old docs?  If so this pattern doesn't quite seem right as `0x0` is valid but `0x06` for example is _not_.

IMO there shouldn't be any `uint` types defined but rather a single `quantity` type as explained here: https://ethereum.org/en/developers/docs/apis/json-rpc/#quantities-encoding

Reply:
Ya that's fair, I could see the advantage of defining the max size for the various result elements, as is you pretty much have to serialize every quantity into a `big.Int` or language equivalent which obviously hurts overall performance.